### ISSUE TITLE
Niveles de usuario

### ISSUE THREAD

**Issue by Yjcr** [Yjcr/catalogo-web#9]
Aunque hay 2 niveles de usuarios ya establecidos (administrador y cliente) aun hace falta la integración de un nivel de usuario correspondiente al gerente de la tienda.

**Comment by DjGuetta** [comment]
No, vale, ¿eso pa qué? El gerente es el mismo que ingresa los juegos mediante el admin que proporciona django. Otra cosa es separarlo del superusuario como tal

**Comment by DjGuetta** [comment]
# Conclusión.
Se requiere agregar e instanciar un nivel de usuario correspondiente a quien está destinada la página; para así, tener un manejo más organizado de los roles de usuario.
Teniendo por fin, un total de **3 roles de usuario**:
#### Superusuario.
#### Admin.
#### Cliente.
